cgroups/fs/hugetlb.go

```diff
@@ -25,5 +40,37 @@ func (s *HugetlbGroup) Remove(d *data) error {
 }
 
 func (s *HugetlbGroup) GetStats(path string, stats *cgroups.Stats) error {
+	pageSizes, err := getHugePageSize()
```

Comment:
I mean to do this once in a `NewHugetlbGroup()` so we only need to do it once per run of libcontainer, not at every stats collection. Or do the hugepage sizes change frequently?